portfolio/src/main/java/com/google/sps/data/BlogMessage.java

```diff
@@ -59,6 +59,10 @@ public String getSender() {
     return messageReplies;
   }
 
+  public void setAdditionalReply(BlogMessage message) {
+    messageReplies.add(message);
```

Comment:
It's not quite clear why you do this. can you add a comment that explains why you add message to messageReplies

Reply:
Actually nevermind